doc/source/myrocks/zenfs.rst

```diff
@@ -115,11 +205,23 @@ Shut down the server and use the following command to backup a ZenFS file system
 
 .. note::
 
-    At this time, a logical backup is the only backup type supported by the Percona Server MyRocks tables located on ``ZenFS``.
+    At this time, a logical backup is the only backup type supported by the *Percona Server for MySQL*. The MyRocks tables are located on ``ZenFS``.
```

Comment:
```suggestion
    At this time, a logical backup is the only backup type supported by *Percona Server for MySQL*. The MyRocks tables are located on ``ZenFS``.
```
I struggle with the second sentence. We should include a better explanation here.

Reply:
Fixed.